plugins/metrics/app/views/metrics/application/index.html.haml

```diff
@@ -28,8 +27,18 @@
         %p
           This is a Prometheus expression browser adapted to Maia. You can use it to discover metrics,
           series and perform ad-hoc queries leveraging all of PromQL's rich query syntax.
-
+        %p
+    - if grafana_available && current_user && current_user.is_allowed?('metrics:application_gaas')
+      %h3
+        = link_to 'Grafana', 'https://github.com/sapcc/maia/blob/master/README.md', target:'_blank'
+      %p
+        Grafana is a system to display Maia metrics. It provides a pre-configured set of dashboards. [ Currently only available in qa-de-1 and eu-ru-1 ].
 %p
-  = link_to :maia, class: "btn btn-primary", target:'_blank' do
+
+  = link_to :maia, class: 'btn btn-primary', target:'_blank' do
     %i.fa.fa-external-link
     Open Maia Dashboard
+  - if grafana_available && current_user && current_user.is_allowed?('metrics:application_gaas')
+    = link_to :gaas, class: 'btn btn-primary', style: 'margin-left: 5px', title: 'only available for qa-de-1 and eu-ru-1', target:'_blank' do
```

Comment:
For the first time click, the user will see an empty new tab and needs to wait until grafana is done. We have the same if an error happened. Than we will see the error page in the new tab. 

Better would be to have an ajax call that is doing the error handling and polling. Thats not a must have, the current version is working well but maybe a good improvement for later ;-)